platforms/shared/configuration/add-new-organization.yaml

```diff
@@ -0,0 +1,19 @@
+---
+#######################################
+# Master Playbook for the setting up and reseting the Blockchain Automation Framework DLT network
+#######################################
+  - hosts: all
+    gather_facts: no
+  - import_playbook: environment-setup.yaml   # First setup environment for each organization
+  - import_playbook: kubernetes-env-setup.yaml # Then deploy Flux for each organization
+    when: reset is undefined or reset == 'false'
+  - import_playbook: "{{ playbook_dir }}/../../hyperledger-fabric/configuration/add-organization.yaml"
+    when: network.type == 'fabric' and (reset is undefined or reset == 'false')
+  - import_playbook: "{{ playbook_dir }}/../../r3-corda/configuration/deploy-network.yaml"
+    when: network.type == 'corda' and (reset is undefined or reset == 'false')
+
+# Reseting the network
+  - import_playbook: "{{ playbook_dir }}/../../r3-corda/configuration/reset-network.yaml"
```

Comment:
no need to add reset here, because reset will be done by site.yaml. This may give the impression that this code removes a node from existing network.